MapleServer2/Types/Player.cs

```diff
@@ -84,7 +84,7 @@ public class Player
 
         public static Player Default(long accountId, long characterId, string name = "SparkmodF")
         {
-            int job = 10;
+            int job = 100;
```

Comment:
What's the purpose of this change?

Reply:
These files aren't suppose to be in the pr.

Reply:
No purpose. ignore this file. Its part of the minor skill tree fix which will have its separate pr after this